From 0f9f572e472bc450052b7fb2d94997f16e85cf15 Mon Sep 17 00:00:00 2001
From: Bahtiar `kalkin-` Gadimov <bahtiar@gadimov.de>
Date: Mon, 21 Aug 2017 13:06:39 +0200
Subject: [PATCH] Show qubes color borders

Signed-off-by: Bahtiar `kalkin-` Gadimov <bahtiar@gadimov.de>
---
 awesomerc.lua | 12 +++++++++---
 1 file changed, 9 insertions(+), 3 deletions(-)

diff --git a/awesomerc.lua b/awesomerc.lua
index dc4babd3..63431c72 100644
--- a/awesomerc.lua
+++ b/awesomerc.lua
@@ -19,6 +19,9 @@ local hotkeys_popup = require("awful.hotkeys_popup")
 -- when client with a matching name is opened:
 require("awful.hotkeys_popup.keys")
 
+local qubes = require("qubes")
+
+
 -- {{{ Error handling
 -- @DOC_ERROR_HANDLING@
 -- Check if awesome encountered an error during startup and fell back to
@@ -529,6 +532,7 @@ client.connect_signal("manage", function (c)
     -- i.e. put it at the end of others instead of setting it master.
     -- if not awesome.startup then awful.client.setslave(c) end
 
+    qubes.manage(c)
     if awesome.startup
       and not c.size_hints.user_position
       and not c.size_hints.program_position then
@@ -552,7 +556,9 @@ client.connect_signal("request::titlebars", function(c)
         end)
     )
 
-    awful.titlebar(c) : setup {
+    qubes.manage(c)
+    awful.titlebar(c, { bg_normal = qubes.get_colour(c),
+                        bg_focus = qubes.get_colour_focus(c) } ) : setup {
         { -- Left
             awful.titlebar.widget.iconwidget(c),
             buttons = buttons,
@@ -586,6 +592,8 @@ end)
 -- @DOC_BORDER@
 client.connect_signal("focus", function(c) c.border_color = beautiful.border_focus end)
 client.connect_signal("unfocus", function(c) c.border_color = beautiful.border_normal end)
+client.connect_signal("focus", function(c) c.border_color = qubes.get_colour_focus(c) end)
+client.connect_signal("unfocus", function(c) c.border_color = qubes.get_colour(c) end)
 -- }}}
 
 -- Use dex to run the xdg autostart files
-- 
2.30.2

